docker-compose.yml

```diff
@@ -210,6 +210,42 @@ services:
       retries: 3
       start_period: 2m
 
+
+  ####################### Charts API #######################
+  charts:
```

Comment:
It will now startup with the `main` profile.  Not certain we want this, but for now it'll work.